main.go

```diff
@@ -0,0 +1,67 @@
+/**
+ *Author: Stefan
+ *Date: 12/10/2019
+ *Last changes: 12/11/2019 01.19
+ *Task: Web server handling any request to it’s address and returning JSON like:
+ */
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+)
+
+//Response struct
+type Response struct {
+	Host       string      `json:"host"`
+	UserAgent  string      `json:"user_agent"`
+	RequestURI string      `json:"request_uri"`
+	Headers    http.Header `json:"headers"`
+}
+
+func jsonHandler(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+
+	p := &Response{
+		Host:       r.Host,
+		UserAgent:  r.UserAgent(),
+		RequestURI: r.RequestURI,
+		Headers:    r.Header,
+	}
+
+	fmt.Fprintln(w, p)
+
+	fmt.Fprintln(w)
```

Comment:
This should return to a client a response formatted as 
```
HTTP response headers

Valid JSON body.
```
As you don't have to write headers by your own, and server will handle it for you, please just drop a JSON as a response body.

Reply:
Done!)